eslint.config.js

```diff
@@ -23,6 +23,7 @@ export default defineFlatConfig([
       '**/.svelte-kit**',
       'packages/scripts/import/old**',
       '**/.kitbook/routes/**',
+      '**/ids-import/**'
```

Comment:
Don't ignore a folder, we want to lint as much as possible

Reply:
Oh, ok! But I did that because of two reasons:

1.  I'm using these packages 
![image](https://github.com/livingtongues/living-dictionaries/assets/43384963/9b149946-6e4b-4b8a-88ab-91beaa9aa125)
So the lint doesn't recognize GoogleAppsScripts types

2. Because the Apps scripts doesn't export and import files as usual. I have a bunch of warnings and errors due to used but not defined variables/functionsand not defined variables/functions.
![image](https://github.com/livingtongues/living-dictionaries/assets/43384963/41fc82d1-19da-4a4b-8c3f-c30dba97ad6c)

Reply:
Let's work on the lint together next meeting. A lot of good code quality rules get ignored when we turn the whole thing off. It's better instead to turn off one particular rule or two in a certain context if it's not helpful.

Reply:
Cool! I'm going to push a few small improvements and finish the rest in our meeting